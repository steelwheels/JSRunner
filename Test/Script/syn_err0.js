const a = a hoge b ;

