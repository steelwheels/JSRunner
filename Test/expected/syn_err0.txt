[31mSyntaxError: Unexpected identifier 'hoge'. Expected ';' after variable declaration.
[0m[31mTypeError: undefined is not an object
[0m